.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.209.6/containers/python-3/.devcontainer/base.Dockerfile
+
```

Comment:
Please remove all github codespace files so we can focus on the relevant code changes in this change set. You are welcome to submit development environment specific changes in  a separate PR. Thanks!